vike/client/client-routing-runtime/renderPageClientSide.ts

```diff
@@ -203,9 +207,18 @@ async function renderPageClientSide(renderArgs: RenderArgs): Promise<void> {
       // Render page view
       await renderPageView(pageContext)
     } else {
-      let res: Awaited<ReturnType<typeof getPageContextFromHooks_isNotHydration>>
+      let res: Awaited<ReturnType<typeof getPageContextFromHooks_isNotHydration>> | PrefetchedPageContext
       try {
-        res = await getPageContextFromHooks_isNotHydration(pageContext, false)
+        const prefetchedPageContext = globalObjectForPrefetchedPageContext?.prefetchedPageContext
+        if (
```

Comment:
We need to check whether it expired.

Reply:
If the user hovered a link 10 minutes ago and now navigates to the url (without hovering), then I think we should not use that 10 minutes old pageContext?

Reply:
I understand. Thanks👍